<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SemiAutomaticClipper</class>
 <widget class="QDialog" name="SemiAutomaticClipper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mean treshold</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Square size</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QSpinBox" name="texStdTresspinBox">
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QSpinBox" name="texMeanTresspinBox">
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Floodfill</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QSpinBox" name="floodUpTresspinBox">
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Upper treshold</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Square size</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QSpinBox" name="floodLoTresspinBox">
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Standard deviation
 treshold</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Lower treshold</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QSpinBox" name="texMeanRGBTresspinBox">
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mean RGB treshold</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QSpinBox" name="texSqSizespinBox">
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Texture search</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="3">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Fill holes</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1" colspan="2">
      <widget class="QSpinBox" name="fillHolesSqSizespinBox">
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="3">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="doNotCutButton">
       <property name="text">
        <string>Do not cut</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="cutButton">
       <property name="text">
        <string>Cut</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
